core/src/main/java/com/freiheit/fuava/simplebatch/processor/BatchJsonLoggingProcessor.java

```diff
@@ -0,0 +1,49 @@
+package com.freiheit.fuava.simplebatch.processor;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import com.freiheit.fuava.simplebatch.fetch.FetchedItem;
+import com.freiheit.fuava.simplebatch.logging.BatchJsonLogger;
+import com.freiheit.fuava.simplebatch.result.Result;
+
+public class BatchJsonLoggingProcessor<Input>
```

Comment:
What is the distinction between BatchJsonLogginProcessor and BatchJsonLoggingFailureProcessor?